common/src/main/java/org/wso2/testgrid/common/ShellExecutor.java

```diff
@@ -38,9 +42,13 @@
 
     private static Logger logger = LoggerFactory.getLogger(ShellExecutor.class);
 
-    private File workingDirectory;
+    private Path workingDirectory;
 
-    public ShellExecutor(File workingDirectory) {
+    public ShellExecutor() {
+        this.workingDirectory = null;
```

Comment:
IMO, this should call the other constructor with current working directory as the argument.

Reply:
This is fixed :+1: